helm-chart/flink-operator/README.md

```diff
@@ -8,7 +8,8 @@ The instructions to install the Flink operator chart:
 
 1. Prepare a Flink operator image. You can either use a released image e.g., `gcr.io/flink-operator/flink-operator:latest` or follow the instructions [here](https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/blob/master/docs/developer_guide.md#build-and-push-docker-image) to build and push an image from the source code.
 
-2. Run the bash script `update_template.sh` to update the manifest files in templates from the Flink operator source repo (This step is only required if you want to install from the local chart repo).
+2. Run the bash script `update_template.sh` to update the manifest files in templates from the Flink operator source repo (This step is only required if you want to install from the local chart repo).  
+If you're installing the operator into a preexisting namespace, make sure to change `flink-operator-system` in the `update_template.sh` script to your namespace.
```

Comment:
That does make more sense. I've added a $NS env var to the script. I haven't documented it since the $IMG is not documented as well, but if you think it's necessary I can add docs for both.

Reply:
Please add them in the doc, thanks!

Reply:
Sure thing, added.